src/CMakeLists.txt

```diff
@@ -208,3 +208,27 @@ add_library(lexy INTERFACE)
 add_library(foonathan::lexy ALIAS lexy)
 target_link_libraries(lexy INTERFACE foonathan::lexy::core foonathan::lexy::file foonathan::lexy::unicode foonathan::lexy::ext)
 
+if(LEXY_ENABLE_INSTALL)
+    set_target_properties(lexy_core PROPERTIES EXPORT_NAME lexy::core)
```

Comment:
Is there no way to automate that? Or export the alias targets instead? I don't like the code duplication.

Reply:
I guess we could define a function that defines the alias targets and sets the property.

Reply:
Can we not export alias targets?

Reply:
I wish we could, but I don't think so.

It is possible to add the alias targets manually to `lexyConfig.cmake.in`, but that is also code duplication.